code/modules/vehicles/_vehicle.dm

```diff
@@ -164,3 +164,15 @@
 	if(trailer && .)
 		var/dir_to_move = get_dir(trailer.loc, newloc)
 		step(trailer, dir_to_move)
+
+/obj/vehicle/attacked_by(obj/item/attacking_item, mob/living/user)
```

Comment:
Good call, I think I had some wires twisted in my head at some point and mistook `/atom/attacked_by` for `/obj/attacked_by,` because `/obj/attacked_by` doesn't exist yet.
Changing it should work fine.